driver/dm_bnorm.cpp

```diff
@@ -31,7 +31,11 @@ static Driver* makeDriver(const std::string& base_arg)
     if(base_arg == "bnorm")
         return new BatchNormDriver<float, double>();
     if(base_arg == "bnormfp16")
+        return new BatchNormDriver<float16, double, float16>();
+    if(base_arg == "bnormfp16fp32")
```

Comment:
"bnormfp16fp32" ? is this correct?

Reply:
Unfortunately, no. We have two different batch normalization kernels: OCL and CK. Both use different data types for their tensors. See the image below

![image](https://github.com/user-attachments/assets/82024627-dca7-47ad-86e5-790a2b0f5dbe)

For example, dScale is fp16 in the OCL kernel and fp32 in the CK kernel. Thus, bnormfp16fp32 is used for CK, while bnormfp16 is used for the OCL kernel's tensor initialization. This is an unsatisfactory solution for now, until we can develop a proper final API.